vowpalwabbit/cb_explore_adf_common.h

```diff
@@ -194,5 +195,13 @@ void cb_explore_adf_base<ExploreType>::finish_multiline_example(
 
   VW::finish_example(all, ec_seq);
 }
+
+template <typename ExploreType>
+inline void cb_explore_adf_base<ExploreType>::save_load(
+    cb_explore_adf_base<ExploreType>& data, io_buf& io, bool read, bool text)
+{
+  data.explore.save_load(io, read, text);
```

Comment:
Looks like I'll need exactly this change for another exploration algorithm I'm working on (save_load hook for cb_explore_adf), so I'll wait for this to merge.

Question: if the ExploreType doesn't call set_save_load ... this is safe?